daemon/services/api/middleware.go

```
@@ -73,3 +75,156 @@ func recoveryMiddleware(next http.Handler) http.Handler {
+
+	return hostport // already just a host
+}
+
+// isLocalhost returns true if the host is a loopback address or "localhost".
+func isLocalhost(host string) bool {
+	switch strings.ToLower(host) {
+	case "localhost", "127.0.0.1", "::1":
+		return true
+	}
+	return false
+}
```

golangci-lint (2.11.4)

[error] File is not properly formatted

(gofmt)